filebeat/crawler/crawler.go

```diff
@@ -6,26 +6,25 @@ import (
 
 	"github.com/elastic/beats/filebeat/input/file"
 	"github.com/elastic/beats/filebeat/prospector"
-	"github.com/elastic/beats/filebeat/spooler"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 )
 
 type Crawler struct {
 	prospectors       []*prospector.Prospector
 	prospectorConfigs []*common.Config
-	spooler           *spooler.Spooler
+	out               prospector.Outlet
```

Comment:
Can you add a comment on this line that "normally" this is the spooler "outlet"? Makes the code much easier to understand.